Packs/MandiantAdvantageAttackSurfaceManagement/Integrations/AttackSurfaceManagement/AttackSurfaceManagement.py

```diff
@@ -0,0 +1,458 @@
+
+    response = client.get_collections_list(project_id)
+
+    output = [{'Name': collection.get('printable_name'),
+               'ID': collection.get('name'),
+               'Owner': collection.get('owner_name')} for collection in response]
+
+    return CommandResults(
+        outputs_prefix='MandiantAdvantageASM.Collections',
+        outputs=output,
+        ignore_auto_extract=True
+    )
+
+
+
+
+def fetch_incidents(client: Client):
+    last_run = demisto.getLastRun()
+
+    last_run, new_issues = fetch_incident_helper(client, last_run)
+
+    demisto.setLastRun(last_run)
+    return new_issues
+
+def get_remote_data_command(client: Client, args: dict):
+    parsed_args = GetRemoteDataArgs(args)
+    if parsed_args.last_update:
+        parsed_date = arg_to_datetime(parsed_args.last_update)
+    if not parsed_date:
+        parsed_date = datetime.now()
+    last_updated = parsed_date
+
+    if not client.project_id:
+        raise DemistoException('Must configure a Project ID to fetch incidents from Mandiant ASM')
+
+    try:
+        masm_id = parsed_args.remote_incident_id
+
+        new_incident_data = helper_create_incident(client.get_issue_details(masm_id), client.project_id)
+
+        notes = client.get_notes('issue', masm_id)
+        notes_entries = []
+        for note in notes:
+            timestamp: datetime = dateutil.parser.parse(note['created_at'])
+            if timestamp.timestamp() > last_updated.timestamp():
+                new_note = {
+                    'Type': EntryType.NOTE,
+                    'Contents': f"{note['note']}\n"
+                                f"{note['created_by_user']['printable_name']}",
+                    'ContentsFormat': EntryFormat.MARKDOWN,
+                    'Note': True,
+                    'Tags': ['note_from_ma_asm']
+                }
+                notes_entries.append(new_note)
+
+        new_incident_data['id'] = masm_id
+
+        return GetRemoteDataResponse(new_incident_data, notes_entries)
+    except Exception as e:
+        raise e
+
+
+def update_remote_system_command(client: Client, args: dict):
+    parsed_args = UpdateRemoteSystemArgs(args)
+
+    if parsed_args.delta:
+        demisto.debug(f'Detected following delta keys: {str(list(parsed_args.delta.keys()))}')
+
+    demisto.debug(parsed_args.inc_status)
+
+    remote_incident_id = parsed_args.remote_incident_id
+
+    if not parsed_args.remote_incident_id or parsed_args.incident_changed:
+        if parsed_args.remote_incident_id:
+            old_incident = client.get_issue_details(remote_incident_id)
+            old_incident.update(parsed_args.delta)
+
+            parsed_args.data = old_incident
+        else:
+            parsed_args.data['createInvestigation'] = True
+
+        demisto.debug(json.dumps(parsed_args.data))
+
+    issue_status = parsed_args.delta.get("runStatus")
+    if issue_status:
+        demisto.debug(f"New issue status: {XSOAR_TO_ASM_STATUS_MAP[issue_status]}")
```

Comment:
```suggestion
        demisto.debug(f"New issue status: {XSOAR_TO_ASM_STATUS_MAP.get(issue_status)}")
```

to avoid any `KeyError`